### DISCUSSION TITLE
音标错误问题

### DISCUSSION THREAD

**Discussion by Maoxin1** [ZuodaoTech/everyone-can-use-english#940] [Q&A]
在发音评估前的页面和发音评估页面中，存在前后同一单词但音标标注不一致的情况。

**Comment by bluechallenge** [comment]
我理解语音评估调用的微软 Azure 服务，这个界面的音标来源和 Enjoy 标注音标时所基于来源是不同的，所以是有可能有差异。从语音评估拿高分的角度，也许得按照得按照语音评估界面给出的音标去尽量做到发音准确。当然，也建议用其它字典，例如 Cambridge Dictionary 或 New Oxford American Dictionary，结合音频去交叉确认。

**Reply by thinkbankcoin** [reply]
我也遇到这个问题了，在不同的平台去核对音标，都会出现不同的结果。

**Reply by bluechallenge** [reply]
如果实在比较困惑的话，也可以参考 [YouGlish 视频](https://youglish.com) 或欧路词典音频 "听力原声例句"，看看单词在不同语境下是怎么发音的。其中，YouGlish 可以指定[不同选项](https://youglish.com/about)，例如美式或英式口音。

**YouGlish**
<img width="742" alt="image" src="https://github.com/user-attachments/assets/089b2663-9f72-4dcb-8dc7-2234bb07257b">

**欧路词典**
<img width="787" alt="image" src="https://github.com/user-attachments/assets/1b08e8a8-4bd0-4b6f-878c-3932def0fdd3">
